### PR TITLE
Sign up page

### PR DESCRIPTION
**Pull request by krate27**
Screenshot (553)

### CODE REVIEW

File: global.css [hunk 1]
```diff
@@ -33,4 +35,10 @@ h2{
 
 h3{
```
Comment (GeetaPogul) [inline global.css:36]: one suggestion is don't use html tag directly. use class name

**Review by GeetaPogul — APPROVED** [review]
do suggestion and good to be merge.
good to me

File: Components-E-Com/login.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            <a href="https://twitter.com/Rohan_415?s=09" class="social-landing-page flex-column"> <i
+                    class="fa-solid fa-heart fa-lg"></i> Wishlist </a>
+            <a href="https://www.linkedin.com/in/rohan-dubey-2854b9201/" class="social-landing-page flex-column"> <i
+                    class="fa-solid fa-cart-shopping fa-lg"></i> Cart </a>
+        </div>
+    </nav>
+    <!-- main-body -->
+    <div class="login-wrapper">
+        <h2 style="letter-spacing: .1rem;" class="spacer-1rem"> Login </h2>
+        <form class="submit-form">
```
Comment (raghav6202) [inline Components-E-Com/login.html:48]: Try to avoid inline styles if possible

**Review by raghav6202 — APPROVED** [review]
LOOKS GOOD TO MERGE👍

**Pull request merged**